essentials/function.mdx

```diff
@@ -325,6 +325,132 @@ print(summarizedResponse.json()['choices'][0]['message']['content'])
 ```
 It´s currently moderate rain in London with a temperature of 8.0 degrees Celsius.
 ```
+### Example 4: `get_email_resume`
+For this example we will use gmail as an email service
+- Objective: Create a resume from your e-mails
+- Parameter: `value` (desired quantity of e-mails), `login` (your e-mail), `password` (your e-mail password)
+- Usage Example: User provides a location and receive local weather information
+
```

Comment:
When you move this to the `Tutorial` page, give an explanation of how it works:

Eg. On the code below, what actually happens is:
1. We send the request to the LLM
2. The LLM sees your emails using IMAP
3. ....